### ISSUE TITLE
browser

### ISSUE THREAD

**Issue by Asexual** [Janilabo/ManaPot-Energizer#2]
Just curious what browser you were using for this simba setup i can get it to work with a random version of ie but just curious if there is a set browser that everyone uses.

**Comment by Janilabo** [comment]
Hey,

I have been using Opera and Firefox.

..but, I think it should work with any browser? Simply target the full browser and not just the game area.
Problem could be the game refreshing, that is when Simba will lose the target of it.

So make sure you get full browser targeted.

**Comment by Janilabo** [comment]
I hope you already noticed that this script will only simply solve the energy restoring part of the game, so if you want to do some macroing, then you'll have to design some scripts for that. :P

**Comment by Asexual** [comment]
Yeah i already have a few in mind your sort of the final piece to the automation 👍 Thanks brother!

**Comment by Janilabo** [comment]
Hey Asexual, there is definitely something going on.. I think there possibly has been update behind the scenes in ManaPot, because it is now mixing white items with black ones - it wasn't mixing em before.

I am working on fixes at the moment. :face_with_head_bandage:

**Comment by Asexual** [comment]
Was just going to let you know today that the bitmap was messing up on white's and blacks

**Comment by Janilabo** [comment]
I think I have found out what the problem is, the button colors has been changed just slightly. :D

Releasing 1.03 in a moment.

**Comment by Janilabo** [comment]
Hey Asexual,

Problem has been fixed, released v1.03 of the script.
Button colors indeed had been updately just slightly, ugh. So it kept mixing white as black because of that

**Comment by Janilabo** [comment]
Added version 1.04, which contains faster image color detection than earlier versions

However, I need to come up with backup scanning for failed images (where default detection isn't going to work).
v1.04 works for roughly 95% of images.

**Comment by Janilabo** [comment]
Actually it is closer to 99% than 95%.. Needs to be 100% though.

**Comment by Janilabo** [comment]
Fixed version of script is now available with backup scan readded. Detection is now 100%.

Backup scan is slow, but it is only for those cases where normal detection doesn't work so, it is better than getting stuck. :) (Takes about 10-30 seconds for it to solve, compared to that normal 1-3 seconds)

NOTE: There was index out of bounds error with earlier versions, I fixed that in 1.06 (didn't notice it earlier), sorry about that.
